ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -20,9 +20,15 @@ const INLINE_STYLE = {
   display: 'inline-block',
 }
 
-const BaseGeneLabelContent = styled(
-  ({ color, label, maxWidth, containerStyle, dispatch, ...props }) => <Label {...props} size="mini" color={color || 'grey'} content={label} />,
-)`
+const BaseGeneLabelContent = styled(({ color, customColor, label, maxWidth, containerStyle, dispatch, ...props }) => {
```

Comment:
It looks like that `GeneLabelContent` is the same as `BaseGeneLabelContent`. Why don't we remove one of them?